gemfile

```diff
@@ -53,6 +53,7 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rails-controller-testing'
   gem 'rspec-rails'
 end
```

Comment:
- [ ] @sam2if Great work implementing the changes requested, however, the gemfile is still in small letters and will generate an error as the previous reviewer showed, I would suggest first changing the name to linked `gems` then commit and the change to `Gemfile` and make another commit, sometimes git refuses to make changes if you are using similar name.